ocs_ci/ocs/ui/views.py

```
@@ -3807,6 +3807,12 @@
         "//li[contains(.,'{namespace}')]",
         By.XPATH,
     ),
+    # "Virtual machines" horizontal tab on the VirtualMachines page (right panel)
+    "virtual_machines_list_tab": (
+        "//button[normalize-space(.)='Virtual machines'] | "
+        "//a[normalize-space(.)='Virtual machines']",
```

Ruff (0.16.0)

[warning] Unparenthesized implicit string concatenation in collection

Did you forget a comma?

(ISC004)